### ISSUE TITLE
Fix instance creation error on VULKAN_SDK >= 1.3.216 by opting-in to extension VK_KHR_PORTABILITY_subset

### ISSUE THREAD

**Issue by a-earthperson** [google/uVkCompute#35]
Related issue posted here: [Encountered VK_ERROR_INCOMPATIBLE_DRIVER](https://vulkan.lunarg.com/doc/sdk/1.3.239.0/mac/getting_started.html).

Proposed fix:
1. Adds the `VK_INSTANCE_CREATE_ENUMERATE_PORTABILITY_BIT_KHR` bit to the `VkInstanceCreateInfo` structure flags.
2. Adds `VK_KHR_portability_enumeration`, `VK_KHR_get_physical_device_properties2` to the instance extensions list.

**Comment by antiagainst** [comment]
Yeah for Apple platforms going through MoltenVK would be requiring that. Please feel free to send out a pull request for it!

**Comment by arjun372** [comment]
> Yeah for Apple platforms going through MoltenVK would be requiring that. Please feel free to send out a pull request for it!

Happy to do it. I have a local branch with the fix working, but I'm unable to push since I don't have write access on remote.
